website/source/docs/enterprise/upgrades/index.html.md

```diff
@@ -6,14 +6,12 @@ description: |-
   Consul Enterprise supports an upgrade pattern that allows operators to deploy a complete cluster of new servers and then just wait for the upgrade to complete.
 ---
 
-# Consul Enterprise Automated Upgrades
+# Automated Upgrades
 
-[Consul Enterprise](https://www.hashicorp.com/consul.html) supports an [upgrade
-pattern](https://learn.hashicorp.com/consul/day-2-operations/autopilot#upgrade-migrations)
-that allows operators to deploy a complete cluster of new servers and then just wait
-for the upgrade to complete. As the new servers join the cluster, server
-introduction logic checks the version of each Consul server. If the version is
-higher than the version on the current set of voters, it will avoid promoting
-the new servers to voters until the number of new servers matches the number of
-existing servers at the previous version. Once the numbers match, Autopilot will
-begin to promote new servers and demote old ones.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) enables the capability of automatically upgrading a Consul cluster to a new
```

Comment:
```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) enables the capability of automatically upgrading a cluster of Consul servers to a new
```
Is it only the servers that are automated or do client upgrades also get automated?

Reply:
Correct, only servers. Good call out here!